feat(cliWrapper): wrap the cucumber CLI to keep the use of custom options (#69)
ekino/veggies: README.md

- [Helpers](#helpers)
    - [**cast**](#cast-helper) [usage](#cast-usage) | [add a type](#add-a-type)
- [Examples](#examples)   

> - [CLI](#cli)   

Reviewer: IMO it should be just after the install and promoted as the standard way to use veggies

Author: Done.

[twitter-badge]: https://img.shields.io/twitter/url/https/github.com/ekino/veggies.svg?style=social

## CLI

Veggies provides a simple CLI allowing the use of custom options to configure the feature it provides.

To make use of it, you can either use one of those commands:
```shell
$ yarn veggies

$ npx veggies

$ ./node_modules/.bin/veggies.js
```

The available options are:

| Option                      | Description                                                                                |
|-----------------------------|--------------------------------------------------------------------------------------------|
| --cleanSnapshots            | remove unused snapshots (not recommended while matching tags)                              |
| -u, --updateSnapshots       | update current snapshots if required                                                       |
| --preventSnapshotsCreation  | a snapshot related step that would create one will fail instead (useful on CI environment) |
| --help                      | prints the veggies CLI help then the cucumber-js CLI help                                  |

> It also lets you pass any CucumberJS CLI options. Please refer to the [CucumberJS CLI documentation](https://github.com/cucumber/cucumber-js/blob/master/docs/cli.md) to see all available options (or use `veggies --help`).

Reviewer: We should tell it uses Cucumber's CLI just after the introduction line 1088, then here :
```suggestion
Please refer to the [CucumberJS CLI documentation](https://github.com/cucumber/cucumber-js/blob/master/docs/cli.md) to see native Cucumber options (or use `veggies --help`).
```

Reviewer: Also the URL should be https://github.com/cucumber/cucumber-js/blob/7.x/docs/cli.md instead of master

Author: I'm not sure about the URL.

As we support cucumber 7 and onward, maybe anyone wanting to see the documentation should refer to the one he/she expects.

WDYT?

Reviewer: Good point, let's just link to master👍

Author: Then I applied your suggestion while keeping the link on master.
